bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "API testing",
+  "type": "collection"
+}
\ No newline at end of file
```

Comment:
It would be better to add a newline at the end of the file to follow good coding practices. [medium]